editor/src/input/input_mapper.rs

```diff
@@ -258,6 +263,17 @@ impl Default for Mapping {
 		];
 
 		let (mut key_up, mut key_down, mut pointer_move, mut mouse_scroll) = mappings;
+		const NUMBER_KEYS: [Key; 10] = [Key::Key0, Key::Key1, Key::Key2, Key::Key3, Key::Key4, Key::Key5, Key::Key6, Key::Key7, Key::Key8, Key::Key9];
```

Comment:
I think we even have a use Keys:* statement above, if not feel free to add that and shorten to [Key1, Key2, …]